app/celery/celery.py

```diff
@@ -55,6 +55,7 @@ def init_app(self, app):
         signals.task_failure.connect(xray_task_failure)
         signals.task_postrun.connect(xray_task_postrun)
         signals.task_prerun.connect(xray_task_prerun)
+        signals.beat_init.connect(xray_task_prerun)
```

Comment:
Signals doc for beat_init:
https://docs.celeryq.dev/en/v5.3.6/userguide/signals.html#beat-init